### DISCUSSION TITLE
I used to be able to play osu!lazer but now it doesn't work

### DISCUSSION THREAD

**Discussion by Pellepunk04** [ppy/osu#15077] [Q&A]
So a long time ago, like back in 2020, I played osu!lazer a couple of times because I liked it. But now for some reason I can't play it.

Here's exactly what happens when I try to play osu!lazer now: I start it, wait for it to load, nothing loads, I press the windows key and then I press on the osu!lazer icon on my taskbar, something "opens" but it's just a small black rectangle at the upper left corner and my mouse disappears, nothing more happens after that and the only thing I can do is try again.

Logs: (Tell me if I need to include another log and I'll try and find it)
[database.log](https://github.com/ppy/osu/files/7335934/database.log)
[performance.log](https://github.com/ppy/osu/files/7335937/performance.log)
[runtime.log](https://github.com/ppy/osu/files/7335939/runtime.log)
[updater.log](https://github.com/ppy/osu/files/7335940/updater.log)

**Comment by bdach** [comment]
There's not much in the logs to go off of. Can you attach a video of how this looks?

**Reply by Pellepunk04** [reply]
https://user-images.githubusercontent.com/69201963/137157309-6d2953f3-f155-4820-9ee1-95351e18ba0b.mp4

This is exactly what I see, and yes I've tried waiting for longer (waited for like 5 minutes once) but it still won't work

**Comment by smoogipoo** [comment]
Check if there are any application logs in event viewer (https://docs.microsoft.com/en-us/host-integration-server/core/how-to-check-the-windows-event-logs-for-errors1).
